themes/default/content/docs/intro/concepts/resources/options/retainOnDelete.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "RetainOnDelete"
+meta_desc: The retain resource option prevents calling the providers Delete function.
+menu:
+  intro:
+    identifier: retainOnDelete
+    parent: options
+    weight: 13
+---
+
+The `retainOnDelete` resource option marks a resource to be retained. If a resource is retained then when
+Pulumi deletes or replaces the resource it will not call through to the resource providers `Delete` method.
```

Comment:
Perhaps add - "As a result, the resource will not be deleted from the backing cloud provider, but will be removed from the Pulumi state".